src/core/Surface/Surface.tsx

```diff
@@ -40,12 +40,12 @@ export type SurfaceProps = {
 } & CommonProps;
 
 /**
- * The surface container allows content to appear elevated through the use of a drop shadow
+ * The Surface container allows content to appear elevated through the use of a drop shadow
  * @example
  * <Surface>Surface Content</Surface>
  * <Surface elevation={2}>Surface Content</Surface>
  */
-export const Surface = (props: SurfaceProps) => {
+export const Surface = React.forwardRef((props: SurfaceProps) => {
```

Comment:
You are not using `ref` here (it comes as 2nd parameter). Check other components, how `forwardRef` syntax looks like or read here:
https://reactjs.org/docs/forwarding-refs.html

Reply:
forwardRef updated. Should be good